flask/boxwise_flask/models/base.py

```diff
@@ -24,13 +60,12 @@ def __str__(self):
 
     @staticmethod
     def get_all_bases():
-        return Base.select().order_by(Base.name)
+        return list(Base.select().order_by(Base.name))
```

Comment:
https://stackoverflow.com/questions/45501197/get-list-of-query-results-in-peewee

This sort of explains why you have list the first place, But I think the model classes should return the required data to the functions that uses it. rather than the functions having to understand how the model classes work.